tensorboard/webapp/app_routing/testing.ts

```diff
@@ -36,6 +36,22 @@ export function buildRoute(routeOverride: Partial<Route> = {}): Route {
   };
 }
 
+export function buildCompareRoute(
+  experimentIds: string[],
```

Comment:
`aliasAndExperimentIds` would be more accurate name.

Reply:
This is not resolved.

Reply:
oops thanks for catching this. forgot to git add.